selene-lib/default_std/luau.yml

```diff
@@ -324,6 +324,12 @@ globals:
     args:
       - type: number
     must_use: true
+  math.lerp:
```

Comment:
Nit: this should be alphabetical order

Reply:
Yes, I wondered if I should make that change, but there's no alphabetical order for the other math entries.

Reply:
Which one isn’t?

Reply:
Just checked and you're right -- that's what I get for quickly scanning at 1am. Will fix in the morning if another maintainer doesn't.